src/components/RestaurantDetailModal/RestaurantDetailModal.jsx

```diff
@@ -1,18 +1,23 @@
 import "./RestaurantDetailModal.css";
 
-function RestaurantDetailModal() {
+function RestaurantDetailModal({ restaurant, onClose }) {
   return (
     <div className="modal modal--open">
-      <div className="modal-backdrop"></div>
-      <div className="modal-container">
-        <h2 className="modal-title text-title">음식점 이름</h2>
+      <div className="modal-backdrop" onClick={onClose}></div>
+      <div className="modal-container" onClick={(e) => e.stopPropagation()}>
```

Comment:
```suggestion
      <div className="modal-container">
```
`e.stopPropagation()` 이 없으면 안되나요??😮

Reply:
오호 지금 로직상 모달 내부 요소가 backdrop의 자식이 아니어서 저 코드가 없더라도 모달 내부를 누르면 모달이 닫히는 일이 일어나지 않을 것 같습니다! 직접 시도해보니 정말 차이가 없네요. 친절한 지적 감사합니다!!! 🤩